terraform/aws/private-cloud/main.tf

```diff
@@ -61,7 +61,7 @@ module "elb" {
 
 # Generate an etcd URL for the cluster
```

Comment:
Cant see the specific case on the docs. This should probably be a https://www.terraform.io/docs/provisioners/null_resource.html anyway